crates/ruff_python_formatter/tests/snapshots/black_compatibility@cases__comments6.py.snap

```diff
@@ -130,18 +130,18 @@ aaaaaaaaaaaaa, bbbbbbbbb = map(list, map(itertools.chain.from_iterable, zip(*ite
 ```diff
 --- Black
 +++ Ruff
-@@ -49,9 +49,7 @@
-     element = 0  # type: int
+@@ -50,8 +50,8 @@
      another_element = 1  # type: float
      another_element_with_long_name = 2  # type: int
--    another_really_really_long_element_with_a_unnecessarily_long_name_to_describe_what_it_does_enterprise_style = (
+     another_really_really_long_element_with_a_unnecessarily_long_name_to_describe_what_it_does_enterprise_style = (
 -        3
 -    )  # type: int
-+    another_really_really_long_element_with_a_unnecessarily_long_name_to_describe_what_it_does_enterprise_style = 3  # type: int
++        3  # type: int
```

Comment:
The style here is different because black doesn't inline the `type: ` comment. Changing the comment to e.g. `ctype` matches Ruff's formatting